generators/server/index.js

```diff
@@ -342,6 +342,126 @@ module.exports = class JHipsterServerGenerator extends BaseBlueprintGenerator {
         return this._writing();
     }
 
+    _postWriting() {
+        return {
+            packageJsonScripts() {
+                const packageJsonStorage = this.createStorage('package.json');
+                const packageJsonConfigStorage = packageJsonStorage.createStorage('config').createProxy();
+                packageJsonConfigStorage.backend_port = this.serverPort;
+                packageJsonConfigStorage.packaging = process.env.JHI_WAR === '1' ? 'war' : 'jar';
+                if (process.env.JHI_PROFILE) {
+                    packageJsonConfigStorage.default_enviroment = process.env.JHI_PROFILE.includes('dev') ? 'dev' : 'prod';
+                }
+            },
+            packageJsonDockerScripts() {
+                const packageJsonStorage = this.createStorage('package.json');
+                const scriptsStorage = packageJsonStorage.createStorage('scripts');
+                const databaseType = this.jhipsterConfig.databaseType;
+                if (databaseType === 'sql') {
+                    const prodDatabaseType = this.jhipsterConfig.prodDatabaseType;
+                    if (prodDatabaseType === 'no' || prodDatabaseType === 'oracle') {
+                        scriptsStorage.set(
+                            'docker:db',
+                            `echo "Docker for db ${prodDatabaseType} not configured for application ${this.baseName}"`
+                        );
+                    } else {
+                        scriptsStorage.set('docker:db', `docker-compose -f src/main/docker/${prodDatabaseType}.yml up -d`);
+                    }
+                } else {
+                    const dockerFile = `src/main/docker/${databaseType}.yml`;
+                    if (databaseType === 'couchbase' || databaseType === 'cassandra') {
+                        scriptsStorage.set({
+                            'docker:db:build': `docker-compose -f ${dockerFile} build`,
+                            'docker:db': `docker-compose -f ${dockerFile} up -d`,
+                        });
+                    } else if (this.fs.exists(this.destinationPath(dockerFile))) {
+                        scriptsStorage.set('docker:db', `docker-compose -f ${dockerFile} up -d`);
+                    } else {
+                        scriptsStorage.set(
+                            'docker:db',
+                            `echo "Docker for db ${databaseType} not configured for application ${this.baseName}"`
+                        );
+                    }
+                }
+
+                const dockerOthers = [];
+                const dockerBuild = [];
+                ['keycloak', 'elasticsearch', 'kafka', 'consul', 'redis', 'memcached', 'jhipster-registry', 'sonar'].forEach(
```

Comment:
plz, remove sonar. It's a special service, which is not needed for starting the application